engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -31,13 +32,38 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
 
   implicit def default = RouteTestTimeout(5.seconds)
 
+  behavior of "REST ENGINE /stats endpoint"
+  it should "return 200 for stats" in {
```

Comment:
toL: Can this be condensed to `"REST ENGINE /stats endpoint" should "return ..."